tests/blas_like/Dot.cpp

```diff
@@ -55,9 +59,39 @@ int main(int argc, char** argv) {
     OutputFromRoot(comm, "Testing Dot");
     TestDot<float, ELEMENT>(m, n, g, print);
     TestDot<float, BLOCK>(m, n, g, print);
+    TestDot<Complex<float>, ELEMENT>(m, n, g, print);
+    TestDot<Complex<float>, BLOCK>(m, n, g, print);
     TestDot<double, ELEMENT>(m, n, g, print);
     TestDot<double, BLOCK>(m, n, g, print);
-  } catch (exception& e) {
+    TestDot<Complex<double>, ELEMENT>(m, n, g, print);
+    TestDot<Complex<double>, BLOCK>(m, n, g, print);
+#if defined(EL_HAVE_QD) && defined(EL_ENABLE_DOUBLEDOUBLE)
```

Comment:
There is no need for the EL_ENABLE macros here (or in any test driver), as they are only used to signal to some of the include macros used for template instantiation in the library.